packages/page-pattern-modal/src/utils/replace-placeholders.ts

```diff
@@ -32,7 +32,7 @@ const replacePlaceholders = (
 };
 
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
-function isObjKey< T >( key: any, obj: T ): key is keyof T {
+function isObjKey< T >( key: any, obj: any ): key is keyof T {
```

Comment:
Thanks for the ping @arcangelini. Unfortunately changing the type of `obj` to `any` means that type `T` no longer gets inferred and the point of the function stops working. What I should have written in the first place is this:

```suggestion
function isObjKey< T extends object >( key: any, obj: T ): key is keyof T {
```

so that TypeScript knows we're allowed to use the `in` keyword on the next line.